ports/libressl/portfile.cmake

```diff
@@ -48,20 +46,21 @@ vcpkg_copy_pdbs()
 # Prepare distribution
 if (tools IN_LIST FEATURES)
     vcpkg_copy_tools(TOOL_NAMES openssl AUTO_CLEAN)
-
-    # Empty directory created during install to house certificated
-    file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/etc/ssl/certs)
 else ()
     # Config and pem files are not installed without the apps
     file(
         INSTALL
-            ${SOURCE_PATH}/apps/openssl/cert.pem
-            ${SOURCE_PATH}/apps/openssl/openssl.cnf
-            ${SOURCE_PATH}/apps/openssl/x509v3.cnf
+            ${SOURCE_PATH}/cert.pem
+            ${SOURCE_PATH}/openssl.cnf
+            ${SOURCE_PATH}/x509v3.cnf

```

Comment:
Interesting that this changed.

Reply:
Yeah hopefully it's not just something with how this release was packaged.